src/main/java/org/folio/roles/service/permission/PermissionOverrider.java

```diff
@@ -26,6 +27,7 @@ public class PermissionOverrider {
   private final ResourceLoader resourceLoader;
   private final PermissionMappingProperties permissionMappingProperties;
 
+  @Cacheable(cacheNames = "permission-mappings")
```

Comment:
To keep this functionality "dynamic", we have to configure cache TTL

Reply:
Permission mappings never change in runtime - one needs to change a file that is packed inside the app JAR to change permission mappings, so it cannot be dynamic anyway.

Reply:
Added cache config.